cfme/tests/openstack/infrastructure/test_host_lifecycle.py

```diff
@@ -46,14 +46,19 @@ def test_scale_provider_down(provider, host, has_mistral_service):
     wait_for(provider.is_refreshed, func_kwargs=dict(refresh_delta=10), timeout=600)
     host.browser.refresh()
     view = navigate_to(host, 'Details')
+    wait_for(lambda: view.entities.summary('Properties').get_text_of('Maintenance Mode') ==
```

Comment:
Required: Add message='Why we are waiting'  for wait_for, and update below

Reply:
Hi,
You mean adding comments on the code that explains the change ?

Reply:
let me give you an example 
https://github.com/ManageIQ/integration_tests/blob/8624a13117640a8007d12642833c2c3ebf8c4db8/cfme/tests/cloud/test_provisioning.py#L279
In case of wait_for failed we will see an exception with this message what we where expecting

Reply:
Thanks for clarification